plugins/upload-assets-to-s3/README.md

```diff
@@ -0,0 +1,79 @@
+# @dotcom-tool-kit/upload-assets-to-s3
+
+Upload files to a configured AWS S3 bucket.
+
+## Installation
+
+Install `@dotcom-tool-kit/upload-assets-to-s3` as a `devDependency` in your app:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/upload-assets-to-s3
+```
+
+Add the plugin to your [Tool Kit configuration](https://github.com/financial-times/dotcom-tool-kit/blob/main/readme.md#configuration):
+
+```yaml
+plugins:
+	- '@dotcom-tool-kit/upload-assets-to-s3'
+```
+
+Install this plugin's tasks:
+
+```sh
+npx dotcom-tool-kit --install
+```
+
+## Options
+| Key | Description | Default value |
+|-|-|-|
+| `accessKeyIdEnvVar` | variable name of the project's aws access key id | no default value - for backwards compatability the plugin falls back to the default value for `accessKeyId` |
+| `secretAccessKeyEnvVar` | variable name of the project's aws secret access key | no default value - for backwards compatability the plugin falls back to the default value for `secretAccessKey` |
+| `accessKeyId` | **DEPRECATED** variable name of the project's aws access key id | 'aws_access_hashed_assets' |
+| `secretAccessKey` | **DEPRECATED** variable name of the project's aws secret access key | 'aws_secret_hashed_assets' |
+| `directory` | the folder in the project whose contents will be uploaded to S3 | 'public' |
+| `reviewBucket` | the development or test S3 bucket | ['ft-next-hashed-assets-preview'] |
+| `prodBucket` | production S3 bucket/s; an array of strings | ['ft-next-hashed-assets-prod', 'ft-next-hashed-assets-prod-us'] |
+| `destination` | the destination folder for uploaded assets. Set to `''` to upload assets to the top level of the bucket | 'hashed-assets/page-kit' |
+| `extensions` | file extensions to be uploaded to S3 | 'js,css,map,gz,br,png,jpg,jpeg,gif,webp,svg,ico,json' |
+| `cacheControl` | header that controls how long your files stay in a CloudFront cache before CloudFront forwards another request to your origin | 'public, max-age=31536000, stale-while-revalidate=60, stale-if-error=3600' |
+
+Example:
+```yml
+'@dotcom-tool-kit/upload-assets-to-s3':
+  '@dotcom-tool-kit/upload-assets-to-s3':
+    accessKeyId: AWS_ACCESS
+    secretAccessKey: AWS_KEY
+    prodBucket: ['ft-next-service-registry-prod', 'ft-next-service-registry-prod-us']
+    reviewBucket: ['ft-next-service-registry-dev']
+    destination: ''
+```
+
+## Testing uploads using the review bucket
```

Comment:
**praise**: it's really useful to have a section like this, taken from your own experience!

Reply:
Well I actually stole some of this from your Cypress README so I'll throw that praise right back at you!